src/test/java/puppy/registration/RegistrationEmptyPasswordFieldTest.java

```diff
@@ -0,0 +1,21 @@
+package puppy.registration;
+
+import org.example.pages.RegistrationPage;
+import org.junit.jupiter.api.Test;
+import puppy.base.BaseTest;
+
+public class RegistrationEmptyPasswordFieldTest extends BaseTest {
+
+    private static final String USERNAME = "Newaccount7";
+    private final RegistrationPage registrationPage = new RegistrationPage();
+
+    @Test
+    public void registrationEmptyPasswordFieldTest() {
+        registrationPage
+                .navigateToRegistrationPage()
+                .enterUsername(USERNAME)
+                .clickRegistrationButton()
```

Comment:
I think you can add a method to clear password fields here, like in the login test (empty credentials login test). Before click to registration button